geometry_msgs/Point[] objectPositions
std_msgs/Int8[] objectTypes
geometry_msgs/Point[] lanePosition0
geometry_msgs/Point[] lanePosition1
geometry_msgs/Point[] lanePosition2
geometry_msgs/Point[] lanePosition3